platformio.ini

```diff
@@ -26,8 +26,9 @@ lib_deps =
   https://github.com/cyberp/DS1337
   https://github.com/practicalarduino/SHT1x
   https://github.com/uChip/MCP342X.git
+  https://github.com/knolleary/pubsubclient
 build_flags = !echo "-DCOOL_FW_VERSION="\'\"$(git describe --tags --always)\"\' "-Wl,-Tesp8266.flash.4m2m.ld -DVTABLES_IN_FLASH
-  -D PIO_FRAMEWORK_ARDUINO_LWIP2_LOW_MEMORY"
+  -D PIO_FRAMEWORK_ARDUINO_LWIP2_LOW_MEMORY -DMQTT_MAX_PACKET_SIZE=3072"
```

Comment:
Hmmm, why wouldn't we leave that to its original value in the source files you've just deleted (`4096`) ?